src/messagingsession/DefaultMessagingSession.ts

```diff
@@ -191,16 +193,21 @@ export default class DefaultMessagingSession implements MessagingSession {
     }
   }
 
+  private retryConnection(): boolean {
+    return this.reconnectController.retryWithBackoff(async () => {
+      try {
+        await this.startConnecting(true);
+      } catch (e) {
```

Comment:
So this behavior is a new thing? Should not the existing code run into this as well?